map_gen/maps/crash_site/commands.lua

```diff
@@ -213,10 +213,10 @@ function Public.control(config)
                 awards.fish_eaten.player = v.name
             end
         end
-
+        local statistics_message = {}
```

Comment:
Lua check is complaining about this
```suggestion
        local statistics_message
```

You don't need to assign it because it's always assigned before use.